src/clang-c-frontend/clang_c_convert.cpp

```diff
@@ -1403,6 +1421,7 @@ bool clang_c_convertert::get_builtin_type(
 
   case clang::BuiltinType::Float16:
   case clang::BuiltinType::Half:
+  case clang::BuiltinType::BFloat16:
```

Comment:
Adding this case here is likely just wrong as far as I know, because afaik the formats of bfloat16 and half differ:
https://en.wikipedia.org/wiki/Bfloat16_floating-point_format#bfloat16_floating-point_format